### DISCUSSION TITLE
Cannot update identity column

### DISCUSSION THREAD

**Discussion by moeSizlak** [jeremyevans/sequel#2166] [General]
I have a sequel model like this, on MSSQL DB (tinysql adapter):

class Code < Sequel::Model(DB1[:HEL_CodeTranslation])
  set_primary_key [:DataSourcePrefix, :Type, :ParentForeignCode, :ForeignCode]
end

The SQL table also has an identity column (ID), this is not part of the primary key.

When I retrieve a record and edit it, the c.save line is throwing the error (Cannot update identity column 'ID'):

        c = Code[prefix, 'OBR', '', foreign_code]
        next if c.nil?
        old_code = c.LocalCode
        c.LocalCode = local_code
        c.SendTEST = 1
        c.SendLIVE = 1
        c.save

How can I tell sequel that the ID column is an identity column and should thus not be updated?

**Comment by jeremyevans** [comment]
Try the skip_saving_columns plugin: https://sequel.jeremyevans.net/rdoc-plugins/classes/Sequel/Plugins/SkipSavingColumns.html

**Comment by moeSizlak** [comment]
That did the trick thank you sir
